.gitignore

```diff
@@ -165,5 +165,5 @@ cython_debug/
 
 # rialto-airflow
 data/
+rialto_airflow/config/*.json
```

Comment:
there is where i've been putting the service account json file while testing ... didn't want it to get inadvertently committed